pages/linux/lxc.md

```diff
@@ -0,0 +1,31 @@
+# lxc
+
+> Manage Linux containers (using the lxd REST API).
+
+- List all containers (does not work on public remotes):
+
+`lxc list [{{private-remote}}:][{{filter}}]`
```

Comment:
A remote is an alias to a URL. One of the first things you learn do is `lxc launch ubuntu:16.04` which creates a new container locally from the default `ubuntu` remote (that's mapped to https://cloud-images.ubuntu.com/releases) and pulls down the most recent `16.04` image. You can make your own remotes and list all that you know about but that might add too many examples for tldr.